site/components/src/test/java/uk/nhs/digital/common/contentrewriters/BrandRefreshContentRewriterTest.java

```diff
@@ -281,7 +281,7 @@ public void rewriteTestForTableWithoutHeader() {
         assertTrue(result.contains("p"));
         assertTrue(result.contains("table"));
         assertFalse(result.contains("cannotsort"));
-        assertFalse(result.contains("caption")); // caption tag removed and text placed in heading
+        assertTrue(result.contains("caption")); // caption tag removed and text placed in heading
```

Comment:
Could we remove the comment here now that it no longer applies?

Reply:
removed